<launch>

  <arg name="uav_name" default="$(optenv UAV_NAME uav)"/>

  <group ns="$(arg uav_name)">
    
    <node name="estimate_location" pkg="uav_detect" type="uav_detect_localize" output="screen" launch-prefix="gdb -ex run --args" >
      
      <rosparam file="$(find mrs_trackers)/config/f550.yaml" />
      <rosparam file="$(find uav_detect)/config/uav_detect.yaml" />
      <param name="uav_name" value="$(arg uav_name)" />

      <remap from="~camera_input" to="mobius_front/image_raw" />
      <remap from="~detections" to="uav_detect/detections" />
    
    </node>
  
  </group>

</launch>
